app/controllers/pseuds_controller.rb

```diff
@@ -146,4 +146,14 @@ def destroy
 
     redirect_to(user_pseuds_url(@user))
   end
+
+  private
+
+  def pseud_params
+    params.require(:pseud).permit(
+      :user_id, :name, :description, :is_default, :icon, :delete_icon, :icon_file_name,
```

Comment:
I am not convinced that the following need to be allowed though:

user_id,  :icon_file_name, :icon_content_type, :icon_file_size, :icon_alt_text, :delta